docs/source/advanced/plugin-configuration.mdx

```diff
@@ -79,12 +77,14 @@ apollo {
 }
 ```
 
+
+> By default, Apollo Kotlin combines all `*.[graphqls|json|sdl]` files in `src/main/graphql/` (Android/JVM projects) or `src/commonMain/graphql/` (multiplatform projects).
+
```

Comment:
Do we need that here? That feels a bit out of place

Reply:
Good catch, I probably pasted by mistake